freqtrade/plugins/pairlist/RemotePairList.py

```diff
@@ -39,12 +39,13 @@ def __init__(self, exchange, pairlistmanager,
                 'for "pairlist.config.pairlist_url"')
 
         self._number_pairs = self._pairlistconfig['number_assets']
-        self._refresh_period = self._pairlistconfig.get('refresh_period', 1800)
+        self._refresh_period: int = self._pairlistconfig.get('refresh_period', 1800)
         self._keep_pairlist_on_failure = self._pairlistconfig.get('keep_pairlist_on_failure', True)
-        self._pair_cache: TTLCache = TTLCache(maxsize=1, ttl=self._refresh_period)
+        self._pair_cache: Optional[TTLCache] = None
```

Comment:
i think this makes it (way) worse 
the TTLCache is still recreated every now and then - but now we have to also check it for None - which seems to add unnecessary complexity.

Reply:
it will have to be recreated in case the remote refresh_period (minimum) changes as theres no way to change a existing cache ttl? So i removed the None Checks again, and let it recreate it once needed. (At start once if its lower than the fetched refresh_period and on additional changes of that)